css/compositing/mix-blend-mode/META.yml

```diff
@@ -0,0 +1,24 @@
+links:
+- product: chrome
+  url: crbug.com/1044742
```

Comment:
This isn't the best bug in the world (its a WPT import please-triage bug), but at least its open and the tests seem related (so sgtm).